src/bodywork/k8s/secrets.py

```diff
@@ -143,6 +143,25 @@ def secret_exists(
         return False
 
 
+def secret_group_exists(namespace: str, group: str) -> bool:
+    """ Does the specified secret group exist
+
+        :param namespace: Kubernetes namespace in which to look for secrets group.
+        :param group: Name of secrets group.
+        :return: True if group exists, otherwise False.
+    """
+    return (
+        len(
```

Comment:
Do you mind unpacking this?

Reply:
I agree it's horrible formatting but this is Black, so it's going to do it every time we run it.

Reply:
It's a little bit better with the conditional on another line.